ddtrace/contrib/pytest/plugin.py

```diff
@@ -132,6 +132,7 @@ def pytest_runtest_protocol(item, nextitem):
             span.set_tag(test.SUITE, item.dtest.globs["__name__"])
         span.set_tag(test.TYPE, SpanTypes.TEST)
         span.set_tag(test.FRAMEWORK_VERSION, pytest.__version__)
+        span.set_tags(_CIVisibility._instance._tags)
```

Comment:
This won't be necessary after #5480 is merged, as the `CIVisibility` service will add its tags to all test spans in a trace through the `CIVisibilityFilter`.

Reply:
Also, I wonder if removing this line will still cause the tests you added to pass, since we are already attaching `CIVisibility._instance._tags` to each root span (which are the only spans we create in the pytest plugin so far)